server/webserver.go

```diff
@@ -984,6 +984,14 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 
 	mux.HandleFunc("/requestTokens", func(w http.ResponseWriter, r *http.Request) {
 		if s.LivepeerNode.Eth != nil {
+
+			nextValidRequest, err := s.LivepeerNode.Eth.NextValidRequest(s.LivepeerNode.Eth.Account().Address)
+			now := time.Now().UnixNano() / int64(time.Second)
```

Comment:
Since the local clock is going to keep progressing even when blockchain time is not progressing it's possible that this check passes, but the request is not yet valid because the block with a timestamp that would make the request valid has not been included yet. Probably safer to just query for the current block timestamp.

Something like:

```
blk, err := s.LivepeerNode.Eth.Backend().GetBlock()
// Handle error
now := blk.Time()
```

Reply:
fixed in 8f0e80d6 but I had to extend the `Backend` interface with an embedded `ChainReader`